pkg/agave/validator.go

```diff
@@ -20,11 +20,25 @@ type KeyPairs struct {
 	VoteAccount string `pulumi:"voteAccount" provider:"secret"`
 }
 
+type Metrics struct {
+	SolanaMetricsURL string `pulumi:"solanaMetricsURL,optional"`
+}
+
+func (agave *Agave) Install() runner.Command {
+	return &InstallCommand{
+		Flags:    agave.Flags,
+		KeyPairs: agave.KeyPairs,
+		Version:  agave.Version,
+		Metrics:  agave.Metrics,
+	}
+}
+
```

Comment:
```suggestion
type Metrics struct {
	URL string `pulumi:"solanaMetricsURL"`
}
```
I think metrics should be optional on Agave but not the url on metrics.

Metrics doesn't require its own install command it is executing the existing one for agave and adding the extra env.